pkgs/development/tools/misc/cli11/default.nix

```diff
@@ -1,6 +1,7 @@
 {
   lib, stdenv,
   fetchFromGitHub,
+  catch2,
   cmake,
   gtest,
   python3,
```

Comment:
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, catch2
, cmake
, gtest
, python3
```
and so on

Reply:
Should i run nixpkgs-fmt on the files while touching them or is that considered unnecessary noise in the pr?